stack.c

```diff
@@ -46,6 +49,32 @@ int mill_max_cached_stacks = 64;
 static int mill_num_cached_stacks = 0;
 static struct mill_slist mill_cached_stacks = {0};
 
+static void *mill_allocstackmem(size_t stack_size) {
+#if MILL_ENABLE_STACK_GUARD
```

Comment:
The question is if a compile time flag is the right way to control this (if it is wanted at all). Goroutine stack size can be controlled at runtime via goprepare().